web/styles/compose.css

```diff
@@ -438,6 +438,14 @@
         background: var(--color-compose-embedded-button-background-hover);
         color: var(--color-compose-send-control-button-interactive);
     }
+
+    &:focus:not(:focus-visible) {
```

Comment:
Hmmmm. I thought that `:focus` and `:focus-visible` are discrete from one another? Is there a reason to explicitly add the `:not(:focus-visible)` expression to the `:focus` selector? Like, is this necessary for specificity reasons? I haven't tested myself yet; just looking at the code.

Reply:
I retested, the focus ring won't appear on navigating via keyboard without that full selector. The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) too mention that:
> The :focus-visible pseudo-class applies while an element matches the [:focus](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus) pseudo-class and the UA ([User Agent](https://developer.mozilla.org/en-US/docs/Glossary/User_agent)) determines via heuristics that the focus should be made evident on the element.

So these aren't discrete and the explicit selector is needed.

Reply:
Hmmmm. That's not how I'm interpreting that, and MDN's own example further down that docs page makes no use of `:not`...I also only see one other instance of `:not` in our codebase (and even that one raises my eyebrow, though for different reasons).

Reply:
If we're redefining all outline related properties in `focus-visible`, then we do not need to be explicit, but if its just 1 property (like color here) we do need the `not...` part to ensure the `focus` styles don't cascade to the `focus-visible` ones

Reply:
Ah, got it. The problem is `outline: none` on focus, and then trying to set `outline-color` on `:focus-visible`. Might it not make sense to just stick with `outline-color`, and set that to `transparent` on a simplified `:focus` selector? That works for me in testing your PR with that adjustment.

Reply:
Sure that works, have switched to `outline-color: transparent` and removed the `not` from the `focus` selector!

Reply:
That does not work in dark mode since specificity causes the `outline-color` defined for these elements in dark mode to override the `transparent` we set here. Even using `!important` doesn't work since it makes the outline transparent even for `focus-visible`.

So I'm reverting to the initial `:focus:not(:focus-visible)` selector with the `outline: none` styling.